### DISCUSSION TITLE
Feedback on SMS code resources page (Office hours: Oct 6, 2020)

### DISCUSSION THREAD

**Discussion by ang-twilio** [twilio-labs/paste#826] [Office hours] [answered]
**The specific questions I have are...**

- Review how I've used components to design the code resources page
- Are there recommended ways of composing this design that I'm not currently doing?
- Are there any other layout recommendations?
- Are there any spacing concerns or recommendations?

**Background:** SMS onboarding flow leads customers to a screen where they can check out code resources. I'm working on using Paste components to create this page. The information shown on this page are a result of customer interviews and usability testing.
**Stage of design:** Discover - Prototype
**Research:** I've previously tested a version of this design as part of a more comprehensive testing of the onboarding designs
**Links to mockups or prototype:** [Invision](https://twilio.invisionapp.com/share/5TYUPRZNQ3P#/432461085_msg-Try-Sms-Resources)

**Comment by kate-porter** [comment]
Thank you @ang-twilio! We'll be sure to review this today before office hours. Just to clarify, you are looking for feedback specifically on the code resources page during office hours, not the full onboarding flow?

**Reply by ang-twilio** [reply]
Yes exactly. I wanted to focus today's conversation on just the code resources page

**Comment by corinne-nagel** [comment, marked as answer]
Hey @ang-twilio ! I left some comments on the Invision about copy but have you tried a layout like this? https://twilio.invisionapp.com/share/DMZ05VIGWYV#/433835467_Numbers

I think removing some of the outlines of the cards will help give some breathing room and organize it so it's not so spread out across the bottom which will help balance things a bit/give better hierarchy to the info. We can chat more in office hours though :)

**Reply by serifluous** [reply]
I also left a few comments on Invision. ^ On the breathing room point, if you do decide to keep the cards, some extra padding inside the cards could help, too.

(I left this comment in there, too, but tagging @TheSisb here for awareness) @ang-twilio, if your illustrations came as SVGs, it'll be super helpful for us if your eng team tries out the SVG-to-React library and lets us know how it goes https://paste.twilio.design/libraries
